fuel-tx/src/transaction/types/mint.rs

```diff
@@ -49,11 +49,16 @@ impl MintMetadata {
 /// by it.
 #[derive(Default, Debug, Clone, Derivative)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(
+    feature = "da-compression",
+    derive(fuel_compression::Compress, fuel_compression::Decompress)
+)]
 #[derive(fuel_types::canonical::Deserialize, fuel_types::canonical::Serialize)]
 #[canonical(prefix = TransactionRepr::Mint)]
 #[derivative(Eq, PartialEq, Hash)]
 pub struct Mint {
     /// The location of the transaction in the block.
+    #[cfg_attr(feature = "da-compression", da_compress(skip))]
```

Comment:
I think we need to skip this field during compression, but during decompression, we need to set it. Maybe it is better to not derive `DecompressibleBy` and implement it via `DecompressionContext`

Reply:
Apparently not needed. Removed `De/CompressionContext` traits  in 9b33143.